lib/maplibre-compose-expressions/src/commonMain/kotlin/dev/sargunv/maplibrecompose/expression/dsl/inputs.kt

```diff
@@ -0,0 +1,20 @@
+package dev.sargunv.maplibrecompose.expression.dsl
+
+import dev.sargunv.maplibrecompose.expression.ast.Expression
+import dev.sargunv.maplibrecompose.expression.ast.FunctionCall
+import dev.sargunv.maplibrecompose.expression.value.FloatValue
+
+/**
+ * Gets the current zoom level. Note that in layer style properties, [zoom] may only appear as the
+ * input to a top-level [step] or [interpolate] (, [interpolateHcl], [interpolateLab], ...)
+ * expression.
```

Comment:
does the linking to these functions continue to work in the IDE, now that these functions are not in the same file anymore?

Reply:
Yup because they're in the same package